modin/experimental/core/execution/native/implementations/omnisci_on_native/test/test_dataframe.py

```diff
@@ -2025,6 +2025,9 @@ def test_shape_hint_detection(self, index):
 
     def test_shape_hint_detection_from_arrow(self):
         at = pyarrow.Table.from_pydict({"a": [1, 2, 3]})
+        # trigger engine initialization for proper `from_arrow` function work
+        _ = pd.DataFrame([])
```

Comment:
This feels a little weird, @vnlitvinov thoughts?

Reply:
Can we just subscribe to the engine here instead?

Reply:
the ideal solution (IMHO) would be to introduce a separate API function that would initialize the engine, instead of subscribing over and over again... but for now I would suggest to be consistent with other places and use `Engine.subscribe()` like we do everywhere else